nodejs/eks/nodegroup.ts

```diff
@@ -904,8 +904,8 @@ export function createManagedNodeGroup(name: string, args: ManagedNodeGroupOptio
         scalingConfig: pulumi.all([
             args.scalingConfig,
         ]).apply(([config]) => {
-            const desiredSize = config && config.desiredSize || 2;
-            const minSize = config && config.minSize || 1;
+            const desiredSize = config && config.desiredSize !== undefined ? config.desiredSize : 2;
+            const minSize = config && config.minSize !== undefined ? config.minSzie : 1;
```

Comment:
Maybe
```suggestion
            const desiredSize = config?.desiredSize ?? 2;
            const minSize = config?.minSize ?? 1;
```

Reply:
thank you @ctison. ts is new to me. this is much nicer.